test/jdk/java/foreign/stackwalk/TestAsyncStackWalk.java

```diff
@@ -47,6 +47,34 @@
  *   TestAsyncStackWalk
  */
 
+/*
+ * @test id=zgc
+ * @requires (((os.arch == "amd64" | os.arch == "x86_64") & sun.arch.data.model == "64") | os.arch == "aarch64") & vm.gc.Z
```

Comment:
I saw ZGC is not supported on some versions of Windows. Is this `vm.gc.Z` requires tag enough to exclude the test on those platforms?